yaml_edit/analysis_options.yaml

```diff
@@ -0,0 +1,12 @@
+# Defines a default set of lint rules enforced for
```

Comment:
Can we make this file just:
https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/analysis_options.yaml